sdk/src/main/res/layout/list_item.xml

```diff
@@ -16,15 +16,32 @@
         android:layout_marginLeft="@dimen/medium_margin"
         tools:src="@drawable/payment_card" />
 
-    <TextView
-        android:id="@+id/text_item_title"
-        style="?android:attr/itemTextAppearance"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/medium_margin"
-        android:layout_marginLeft="@dimen/medium_margin"
-        android:layout_weight="1"
-        tools:text="Credit Card" />
+    <LinearLayout
+            android:orientation="vertical"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/medium_margin"
+            android:layout_marginLeft="@dimen/medium_margin"
+            android:layout_weight="1"
+            android:id="@+id/text_with_subtext_view">
+        <TextView
+                android:id="@+id/text_item_title"
+                style="?android:attr/itemTextAppearance"
+                android:layout_width="match_parent"
+                android:layout_height="0dp"
+                android:layout_marginStart="@dimen/medium_margin"
+                android:layout_marginLeft="@dimen/medium_margin"
+                android:layout_weight="3"
+                tools:text="Credit Card" tools:ignore="NestedWeights"/>
+        <TextView
+                android:id="@+id/subtext_item_title"
+                style="?android:attr/itemTextAppearance"
```

Comment:
This style duplicated with the title text. So, subtitle might looks the same with the title text.

```suggestion
                style="?android:attr/textAppearanceListItemSecondary"
```

However, please test this with different themes to test how's it look like.
 https://developer.android.com/reference/android/R.attr#textAppearanceListItemSecondary

Reply:
@nuxzero Your suggestion already looks good, thanks.